docs/source/services/timelock_service/installation.rst

```diff
@@ -118,5 +114,5 @@ Add client(s) to Timelock
 
 14. Restart each client.
 
-15. To verify that the migration worked correctly, get a fresh timestamp for each client from the Timelock server.
+15. (Optional) To verify that the migration worked correctly, get a fresh timestamp for each client from the Timelock server.
```

Comment:
Should we add that if you use the CLI to fetch timestamp before restarting the client you will get 1 as the timestamp. There was a question in the help room regarding this.